bin/maf/generate_ss

```diff
@@ -54,74 +54,80 @@ def generate_ss_commands(dbfiles=None, pops=None, start_mjd=60218.0, split=False
 
     runs = [os.path.split(file)[-1].replace(".db", "") for file in dbfiles]
     runs = [run for run in runs if "tracking" not in run]
+    if not split:
+        output_file = open("ss_script.sh", "w")
```

Comment:
I wasn't sure if it was helpful for your use to have a single file or not. It sounded like you probably would have found multiple names more useful, so you could run in parallel. 
If this is fine, sounds good.